packages/widgets/Source/BaseLayerPicker/createDefaultTerrainProviderViewModels.js

```diff
@@ -32,7 +32,7 @@ function createDefaultTerrainProviderViewModels() {
         "High-resolution global terrain tileset curated from several datasources and hosted by Cesium ion",
       category: "Cesium ion",
       creationFunction: function () {
```

Comment:
Doesn't the code that calls `creationFunction` need to be updated to handle promises? Or does that all "just work"? For example, what happens if the `creationFunction` Promise is rejected?